[2023-05-01T14:25:03.755+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: space_missions_launches_to_gcp_dag.space_missions_launches_from_gcs_to_bq_task manual__2023-05-01T14:24:59.461485+00:00 [queued]>
[2023-05-01T14:25:03.759+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: space_missions_launches_to_gcp_dag.space_missions_launches_from_gcs_to_bq_task manual__2023-05-01T14:24:59.461485+00:00 [queued]>
[2023-05-01T14:25:03.760+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T14:25:03.760+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-01T14:25:03.760+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-01T14:25:03.769+0000] {taskinstance.py:1309} INFO - Executing <Task(GCSToBigQueryOperator): space_missions_launches_from_gcs_to_bq_task> on 2023-05-01 14:24:59.461485+00:00
[2023-05-01T14:25:03.774+0000] {standard_task_runner.py:55} INFO - Started process 10639 to run task
[2023-05-01T14:25:03.776+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'space_missions_launches_to_gcp_dag', 'space_missions_launches_from_gcs_to_bq_task', 'manual__2023-05-01T14:24:59.461485+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/space_missions_launches_to_gcp.py', '--cfg-path', '/tmp/tmpkj9bn1bh']
[2023-05-01T14:25:03.776+0000] {standard_task_runner.py:83} INFO - Job 53: Subtask space_missions_launches_from_gcs_to_bq_task
[2023-05-01T14:25:03.815+0000] {task_command.py:389} INFO - Running <TaskInstance: space_missions_launches_to_gcp_dag.space_missions_launches_from_gcs_to_bq_task manual__2023-05-01T14:24:59.461485+00:00 [running]> on host e47d5c97588c
[2023-05-01T14:25:03.860+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=viktoria.k
AIRFLOW_CTX_DAG_ID=space_missions_launches_to_gcp_dag
AIRFLOW_CTX_TASK_ID=space_missions_launches_from_gcs_to_bq_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-01T14:24:59.461485+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-01T14:24:59.461485+00:00
[2023-05-01T14:25:03.867+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-05-01T14:25:03.871+0000] {gcs_to_bigquery.py:382} INFO - Using existing BigQuery table for storing data...
[2023-05-01T14:25:03.902+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 383, in execute
    self.configuration = self._use_existing_table()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 629, in _use_existing_table
    "schema_update_options is only "
ValueError: schema_update_options is only allowed if write_disposition is 'WRITE_APPEND' or 'WRITE_TRUNCATE'.
[2023-05-01T14:25:03.908+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=space_missions_launches_to_gcp_dag, task_id=space_missions_launches_from_gcs_to_bq_task, execution_date=20230501T142459, start_date=20230501T142503, end_date=20230501T142503
[2023-05-01T14:25:03.916+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 53 for task space_missions_launches_from_gcs_to_bq_task (schema_update_options is only allowed if write_disposition is 'WRITE_APPEND' or 'WRITE_TRUNCATE'.; 10639)
[2023-05-01T14:25:03.948+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-01T14:25:03.959+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
